### PR TITLE
SUPESC-811 Added compatibility with module Country version 4.

### PR DESCRIPTION
**Pull request by olhalivitchuk**
- Developer(s): @olhalivitchuk 

- Ticket: https://spryker.atlassian.net/browse/SUPESC-811

- PR Overview: https://release.spryker.com/release/pull-request-eco/Computop/48

- merge: squash

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Computop              | minor                 |                       |

-----------------------------------------

##### Change log

Improvements

- Added PHP 8.1 support.
- Adjusted `composer.json` to allow for a new `Country` module major.

### CODE REVIEW

**Review by geega — APPROVED** [review]

**Review by asmarovydlo — COMMENTED** [review]
1. Please mention PHP version update.
2. Please take a look to adjusted changelogs

**Review by asmarovydlo — APPROVED** [review]

File: tests/_data/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (asmarovydlo) [inline tests/_data/.gitignore:2]: Please add line break.

File: src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/CancelPlugin.php [hunk 2]
```diff
@@ -13,6 +13,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/CancelPlugin.php:17]: Query containers are outdated, please remove the line.

File: src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/CapturePlugin.php [hunk 3]
```diff
@@ -13,6 +13,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/CapturePlugin.php:17]: Query containers are outdated, please remove the line.

File: src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/EasyCreditAuthorizePlugin.php [hunk 4]
```diff
@@ -13,6 +13,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/EasyCreditAuthorizePlugin.php:17]: Query containers are outdated, please remove the line.

File: src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/RefundPlugin.php [hunk 5]
```diff
@@ -13,6 +13,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/RefundPlugin.php:17]: Query containers are outdated, please remove the line.

File: src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/AuthorizePlugin.php [hunk 6]
```diff
@@ -13,6 +13,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\ComputopConfig getConfig()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Plugin/Oms/Command/AuthorizePlugin.php:17]:
```suggestion
```

File: src/SprykerEco/Zed/Computop/Communication/Controller/GatewayController.php [hunk 7]
```diff
@@ -14,6 +14,9 @@
 
 /**
  * @method \SprykerEco\Zed\Computop\Business\ComputopFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface getQueryContainer()
```
Comment (asmarovydlo) [inline src/SprykerEco/Zed/Computop/Communication/Controller/GatewayController.php:17]:
```suggestion
```

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**